ksqldb-engine/src/main/java/io/confluent/ksql/physical/pull/PullPhysicalPlan.java

```diff
@@ -192,7 +190,6 @@ public QueryId getQueryId() {
    */
   public enum RoutingNodeType {
     SOURCE_NODE,
-    REMOTE_NODE,
-    UNKNOWN
```

Comment:
And finally, ksql is never uncertain whether it's answering a query for a user or another ksql node.